src/CMakeLists.txt

```diff
@@ -137,6 +137,16 @@ SET_DIRECTORY_PROPERTIES(PROPERTIES ADDITIONAL_MAKE_CLEAN_FILES "${LMMS_RCC_OUT}
 
 IF(LMMS_BUILD_WIN32)
 	SET(EXTRA_LIBRARIES "winmm")
+
+	# Find Qt's qwindows.dll and install it.
```

Comment:
The standard way to do that in CMake is:
```cmake
IF(LMMS_BUILD_WIN32)
	INSTALL(FILES $<TARGET_FILE:Qt5::QWindowsIntegrationPlugin> DESTINATION platforms)
ENDIF()
```
(from https://github.com/lukas-w/lmms/commit/4d5cd9a8cc2d170c09b8259436bebc363f397c66?diff=split#diff-e7bc98360bbad3a6a2c524ae5bc94b8fR38)